packages/webapp/src/pages/GettingStarted/Show.tsx

```diff
@@ -0,0 +1,204 @@
+import { IconChevronRight, IconLockOpen2, IconPencil, IconPlayerPlay, IconRefresh, IconTool } from '@tabler/icons-react';
+import { LeftNavBarItems } from '../../components/LeftNavBar';
+import Button from '../../components/ui/button/Button';
+import { Tag } from '../../components/ui/label/Tag';
+import DashboardLayout from '../../layout/DashboardLayout';
+import { useAnalyticsTrack } from '../../utils/analytics';
+import { Helmet } from 'react-helmet';
+import { useScript } from '@uidotdev/usehooks';
+import { useEffect, useState } from 'react';
+import { cn } from '../../utils/utils';
+import { globalEnv } from '../../utils/env';
+import { apiPatchOnboarding } from '../../hooks/useOnboarding';
+import { useStore } from '../../store';
+
+let ytLoaded = false;
+export const GettingStarted: React.FC = () => {
+    const analyticsTrack = useAnalyticsTrack();
+    const env = useStore((state) => state.env);
+    const [hasVideo, setHasVideo] = useState(false);
+
+    useEffect(() => {
+        // The API will call this function when page has finished downloading
+        // @ts-expect-error yes I want this
+        window.onYouTubeIframeAPIReady = () => {
+            ytLoaded = true;
+        };
+    }, []);
+
+    useScript('https://www.youtube.com/iframe_api');
+
+    const triggerVideo = () => {
+        if (hasVideo) {
+            return;
+        }
+        if (!ytLoaded) {
+            // adblock
```

Comment:
adblock?

Reply:
If YouTube does not load it's most of the time adblocked (third party script blocked), I didn't really bothered finding a solution. Most of the time people add youtube the allowlist anyway